src/stories/6-Modal.stories.js

```diff
@@ -1,36 +1,80 @@
 import React from "react";
 import Modal from "../components/Modal";
-// import { action } from "@storybook/addon-actions";
+import DurationBar from "../components/DurationBar";
+import RangeBar from "../components/RangeBar";
+import { action } from "@storybook/addon-actions";
+import styled from "styled-components";
+import Badge from "../components/Badge";
 
 export default {
   title: "Modal-Components"
 };
 
+const BadgeContainer = styled.div`
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+`;
+
+const Title = styled.h4`
+  color: grey;
+  font-family: Arial, Helvetica, sans-serif;
+  align-self: left;
+`;
+
+const Line = styled.hr`
+  color: grey;
+  width: 80%;
+`;
+
 export const Modalize = () => (
   <>
     <Modal />
   </>
 );
-// export function Show() {
-//   return (
-//     <Modal onAccept={action("accept")} onClose={action("close")}>
-//       <ModalTitle>Rate our service!</ModalTitle>
-//       <ModalSection>Rating</ModalSection>
-//     </Modal>
-//   );
-// }
+export function Show() {
+  return (
+    <Modal onAccept={action("accept")} onClose={action("close")}>
+      <Title>Choose Contract Duration</Title>
+      <DurationBar />
+      <Title>Choose Price</Title>
+      <RangeBar />
+      <Line />
+      <Title>Select Folowing</Title>
+      <BadgeContainer>
+        <Badge>Courses</Badge>
```

Comment:
Use an array and build these badges with the ".map" function. If you make it work in your story, it will work in your project as well. Thats one of the big benefits of Storybook :)